src/pages/dashboard/widgets/alarms/index.tsx

```diff
@@ -80,11 +80,11 @@ const alarmsItems: TableProps<Item>['items'] = [
 function AlarmsContent() {
   return (
     <Table
+      enableKeyboardNavigation={true}
```

Comment:
Why all these changes to the order of this prop?

Reply:
That is due to the use of script to copy changes between internal and external demos. The changes to keyboard nav were manual so that's why there is a diff.